lib/parseSty.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+var balanced = require("balanced-match");
+
+var declarationCommands = ["\\newcommand",
+                           "\\renewcommand",
+                           "\\DeclareRobustCommand",
+                           "\\defSpecFun"];
+
+var findBracketSections = module.exports.findBracketSections = function(string, sections) {
+    if (typeof sections === "undefined") {
+        sections = [];
+    }
+    var bracket = balanced("[", "]", string);
+    var brace = balanced("{", "}", string);
+    if (typeof brace !== "undefined" && (typeof brace !== undefined || brace.start < bracket.start)) {
```

Comment:
If I understand the code right the logic is, if either of bracket or brace is undefined use the other. If neither is undefined use the first one, correct?